app/Console/Commands/Sync.php

```diff
@@ -44,12 +45,17 @@ class Sync extends Command
 	public function __construct()
 	{
 		// Fill signature with default values from user configuration
-		$this->signature = sprintf(
-			$this->signature,
-			Configs::getValueAsString('delete_imported', '0'),
-			Configs::getValueAsString('import_via_symlink', '0'),
-			Configs::getValueAsString('skip_duplicates', '0')
-		);
+		try {
+			$this->signature = sprintf(
+				$this->signature,
+				Configs::getValueAsString('delete_imported'),
+				Configs::getValueAsString('import_via_symlink'),
+				Configs::getValueAsString('skip_duplicates')
+			);
+		} catch (ConfigurationKeyMissingException) {
+			// This is necessary other wise we can't even run Composer install
```

Comment:
```suggestion
			// This is necessary, otherwise we can't even run Composer install
```

I'm curious why `composer install` doesn't work without it? Doesn't `composer install` simply download the dependencies based on `composer.lock`? How would the `lychee:sync` console command be affected by any of this?

Or is it that these particular objects are instantiated no matter what when invoking the composer, even if they are not being used?